services/backend/src/database/migrations/20200810152146-optional-interested-in-remote/schema.prisma

```diff
@@ -485,6 +486,7 @@ model User {
   telephone            String?
   cellphone            String?
   manager              String?
+  description          String?
```

Comment:
Why is this file modified, this seem to be an old migration?

Reply:
I think backend changes got merged incorrectly at some point. This schema was missing description even though it was added after the description card change, so there were unexpected steps being included in the migration

Reply:
Ok sounds good, just hope it won't frig up the migrations on the server :crossed_fingers: